R/api.R

```diff
@@ -84,9 +84,11 @@ metadata <- function() {
   step <- response$parameters[[hospital_capacity_idx]]$step
   # setNames to get json object not array
   hospital_capacities <- lapply(
-    setNames(country_names, country_names),
-    function(country) {
-      default <- daedalus::daedalus_country(country)$hospital_capacity
+    setNames(country_codes, country_codes),
+    function(country_code) {
+      idx <- match(country_code, country_codes)
+      country_name <- country_names[[idx]]
+      default <- daedalus::daedalus_country(country_name)$hospital_capacity
```

Comment:
There must be a nicer way of getting the index here..?

Reply:
```suggestion
    setNames(country_names, country_codes),
    function(country_name) {
      default <- daedalus::daedalus_country(country_name)$hospital_capacity

```